res/style.css

```diff
@@ -456,10 +457,12 @@ body, #root, .profileViewer {
 
 .grippyMoveRange {
   flex: 1;
+  cursor: -webkit-grab;
   cursor: grab;
 }
 
 .grippyMoveRange.dragging {
+  cursor: -webkit-grabbing;
```

Comment:
Thanks for adding this, I had an item on my todo list called "Does Chrome have a grab cursor?" which I can now cross out :)